packages/db/src/meta/graph/types.ts

```diff
@@ -2,7 +2,7 @@ import { logger } from "@truffle/db/logger";
 const debug = logger("db:graphql:types");
 
 import type * as graphql from "graphql";
-import type { IResolvers } from "graphql-tools";
+import { IResolvers } from "@graphql-tools/utils";
```

Comment:
In general, we should `import type` wherever possible - normal imports will naturally generate JS code, but if you only import the type, then the JS gets thrown out! (which is nice if you don't need it.)

Also, if you only ever `import type` a package, it typically means you can reduce it to a devDependency, which is also nice.

```suggestion
import type { IResolvers } from "@graphql-tools/utils";
```

Reply:
:+1: